### PR TITLE
story/BODS-2343

### PR DESCRIPTION
**Pull request by pritish-ranjan01**
This PR includes changes for:-

https://kpmgengineering.atlassian.net/jira/software/c/projects/BODS/boards/1797?selectedIssue=BODS-2343

### CODE REVIEW

File: transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py [hunk 1]
```diff
@@ -195,6 +199,37 @@ def df_to_serviced_organisations(
...
+            vehicle_journey_id=record["id"], day_of_week=record["days_of_week"]
+        )
+
+
+def df_to_serviced_org_vehicle_journey(df: pd.DataFrame) -> Iterator[VehicleJourney]:
+    for record in df.to_dict("records"):
+        yield ServicedOrganisationVehicleJourney(
+            vehicle_journey_id=record["vehicle_journey_id"],
+            serviced_organisation_id=record["serviced_org_id"],
+            operating_on_working_days=record["operational_y"],
```
Comment (alvak1) [inline transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py:229]: Could we had a custom suffix

File: transit_odp/timetables/tasks.py [hunk 2]
```diff
@@ -72,13 +72,13 @@ def task_dataset_pipeline(self, revision_id: int, do_publish=False):
         args = (task.id,)
         jobs = [
             task_dataset_download.signature(args),
-            task_scan_timetables.signature(args),
-            task_timetable_file_check.signature(args),
-            task_timetable_schema_check.signature(args),
-            task_post_schema_check.signature(args),
+            # task_scan_timetables.signature(args),
```
Comment (alvak1) [inline transit_odp/timetables/tasks.py:75]: Incorrectly pushed to git

File: transit_odp/pipelines/pipelines/dataset_etl/utils/transform.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import pandas as pd
+import numpy as np
```
Comment (alvak1) [inline transit_odp/pipelines/pipelines/dataset_etl/utils/transform.py:2]: This can be removed

File: transit_odp/timetables/loaders.py [hunk 4]
```diff
@@ -152,35 +172,175 @@ def load_vehicle_journeys(self):
...
+                [
+                    "file_id",
+                    "service_code",
+                    "vehicle_journey_code",
+                    "serviced_org_ref",
+                    "operational",
+                ]
+            ]
+
+            merged_df = pd.merge(
```
Comment (alvak1) [inline transit_odp/timetables/loaders.py:321]: Can `merged_df` be renamed into `df_serviced_org_to_op_profile`

File: transit_odp/timetables/loaders.py [hunk 5]
```diff
@@ -152,35 +172,175 @@ def load_vehicle_journeys(self):
...
+                operating_profiles,
+                serviced_organisations,
+                on=["file_id", "serviced_org_ref"],
+                how="inner",
+            )
+            merged_df.drop_duplicates(inplace=True)
+
+            output_merged_df = pd.merge(
```
Comment (alvak1) [inline transit_odp/timetables/loaders.py:329]: Can `output_merged_df` be renamed into indicate the contents of the dataframe?

**Review by alvak1 — COMMENTED** [review]
added comments

File: transit_odp/timetables/loaders.py [hunk 6]
```diff
@@ -281,17 +280,27 @@ def load_operating_profiles(self, vehicle_journeys):
...
+                            "vehicle_journey_code",
+                            "day_of_week",
+                            "service_code",
+                            "file_id",
+                        ]
+                    ],
+                    on=["file_id", "service_code", "vehicle_journey_code"],
+                    how="inner",
+                )
+                .dropna(subset=["day_of_week"])
```
Comment (alvak1) [inline transit_odp/timetables/loaders.py:299]: Could we remove this line as we are using query to filter out empty values

Comment (pritish-ranjan01) [reply transit_odp/timetables/loaders.py]: Done

**Review by alvak1 — APPROVED** [review]
Approve

**Pull request merged**